temporalcli/commands.gen.go

```diff
@@ -473,10 +475,35 @@ func NewTemporalActivityFailCommand(cctx *CommandContext, parent *TemporalActivi
 	return &s
 }
 
+type ActivityReferenceOptions struct {
+	ActivityId   string
+	ActivityType string
+}
+
+func (v *ActivityReferenceOptions) buildFlags(_ *CommandContext, f *pflag.FlagSet, activityType string, onlyWithoutQuery bool) {
```

Comment:
You can't add code to this code-generated file, it'd get removed on regen. Rather, add in another file (or maybe better, do not add methods to generated structures beyond the `run` method needed on command, but make simple utility functions that accept them).